qutip/tests/test_operators.py

```diff
@@ -33,34 +33,43 @@
 
 import numbers
 import numpy as np
-from numpy.testing import assert_equal, run_module_suite
 import pytest
-
 import qutip
-from qutip import (jmat, basis, destroy, create, displace, qzero, qeye,
-                    num, squeeze, charge, tunneling)
+
+
+N = 5
 
 
 def test_jmat_12():
-    "Spin 1/2 operators"
-    spinhalf = jmat(1 / 2.)
+    spinhalf = qutip.jmat(1 / 2.)
 
     paulix = np.array([[0.0 + 0.j, 0.5 + 0.j], [0.5 + 0.j, 0.0 + 0.j]])
     pauliy = np.array([[0. + 0.j, 0. - 0.5j], [0. + 0.5j, 0. + 0.j]])
     pauliz = np.array([[0.5 + 0.j, 0.0 + 0.j], [0.0 + 0.j, -0.5 + 0.j]])
```

Comment:
These aren't actually the definitions of the Pauli matrices??  (I know you didn't add this code - perhaps we could change them to be the actual definitions, and adjust the tests accordingly?)